tests/src/events.producer.ts

```diff
@@ -14,5 +14,7 @@ export class EventsProducer implements OnApplicationBootstrap {
     this.eventEmitter.emit('test.event', TEST_EVENT_PAYLOAD);
     this.eventEmitter.emit('multiple.event', TEST_EVENT_MULTIPLE_PAYLOAD);
     this.eventEmitter.emit('string.event', TEST_EVENT_STRING_PAYLOAD);
+    this.eventEmitter.emit('stacked1.event', TEST_EVENT_PAYLOAD);
+    this.eventEmitter.emit('stacked2.event', TEST_EVENT_PAYLOAD);
```

Comment:
Went with `stacked` because `multiple` is already used, and `array` would suggest payload type. Rather unimportant.